### DISCUSSION TITLE
Short bursts of audio? how to pause audio stream?

### DISCUSSION THREAD

**Discussion by shayo** [pschatzmann/ESP32-A2DP#413] [Q&A] [answered]
What is the recommended way to send short bursts of audio and nothing in between?
Should I just return 0 in get_data_channels when I don't want to send any data? or is there a way to stop and resume?

**Comment by pschatzmann** [comment, marked as answer]
Starting and stopping will take quite some time.
You could try the AVRC pause() and play() commends.
But the by far easiest solution is just to send an array of 0 values when you don't want to hear any data.
